packages/components/src/mobile/bottom-sheet/stepper-cell/stepper.android.js

```diff
@@ -37,6 +37,7 @@ function Stepper( {
 	return (
 		<View style={ styles.container }>
 			<TouchableOpacity
+				testID={ 'Increment' }
```

Comment:
I think this button is the one that decrements so we should move the `testID` to the one in line. 59.

Reply:
🤦‍♂️ Thanks for pointing that out, @fluiddot!